backends/cadence/README.md

```diff
@@ -4,6 +4,10 @@
 - HiFi Audio
 - ...
 
+## Setup (Internal Only)
```

Comment:
should we put this in OSS at all?

Reply:
Tentatively yes, because it's instruction for cadence oss but exposes nothing extra

Reply:
maybe I'm reading this wrong, we have an internal link and the arvr and nfs paths there... that's all internal right?

Reply:
Correct. Just rethought I will remove these lines